data_taking_scripts/motor_control.py

```diff
@@ -10,19 +23,6 @@ def cm_to_inch(dist):
 def inch_to_cm(dist):
     return dist*2.54
 
-# names should be exactly like 'curved_mirror', 'top_dielectric_plate', 'bottom_dielectric_plate'
-motors_to_move = ['curved_mirror', 'bottom_dielectric_plate', 'top_dielectric_plate']
-#motors_to_move = ['curved_mirror']
-
-average_enable = 1 # 1 or 0
-averages = 16 # value doesn't matter is averable_enable = False
-narrow_scan = True
-wide_scan_start_freq = 15e9
-wide_scan_stop_freq = 18e9
-narrow_scan_span = 400e6
-sec_wait_for_na_averaging = 2
-num_plates = 4
-
 initial_mirror_holder_spacing = float(input('Enter initial mirror holder spacing (in cm): '))
```

Comment:
this is defined in configure.yaml, but then it is overwritten here.

Reply:
you are right. I removed it